client/api.py

```diff
@@ -4,18 +4,21 @@
 
 from flask import Flask
 from flask import request
+from flask.ext.cors import CORS
```

Comment:
I realized the issue I was having with gunicorn was some venv issue so I put it back in